### PR TITLE
More opcodes for AVR anal plugin

### PR DESCRIPTION
**Pull request by killabytenow**
New ops BREAK, BST, CBI, COM, CPSE, DEC, LD, LDD, INC, SBRC/S, STD, EIJMP, EICALL and ELPM.

Heavy bugfixing and refactorization.

Emit cycles warning only one time.

working on FMUL

### CODE REVIEW

File: libr/anal/p/anal_avr.c [hunk 1]
```diff
@@ -1,10 +1,11 @@
-/* radare - LGPL - Copyright 2011-2016 - pancake, Roc Valles, condret */
+/* radare - LGPL - Copyright 2011-2016 - pancake, Roc Valles, condret, killabyte */
 
 #if 0
 http://www.atmel.com/images/atmel-0856-avr-instruction-set-manual.pdf
 https://en.wikipedia.org/wiki/Atmel_AVR_instruction_set
 #endif
 
+#define _GNU_SOURCE
```
Comment (radare) [inline libr/anal/p/anal_avr.c]: is this necessary? i hate gnuisms. try to avoid it if not necessary pls

**Comment by radare** [comment]
the whole pr lgtm, waiting for travis to merge

File: libr/anal/p/anal_avr.c (hunk 1 above)
Comment (killabytenow) [inline libr/anal/p/anal_avr.c]: Seems that strcasestr() needs this directive...

radare2 has implemented some helper to avoid using that?

**Comment by radare** [comment]
strcasestr is a gnuism that doesnt exists on windows or osx, so pls dont use that and check libr/util/str.c

r_str_casestr() should do the job

**Comment by killabytenow** [comment]
strcasestr() and GNU have been removed

**Pull request merged**